src/setOfPeople.js

```diff
@@ -37,8 +37,20 @@
  *
  * @return {Set}
  */
-function setOfPeople(people) {
-  // write code here
+function setOfPeople(people = []) {
+  const uniquePeople = [];
+  const whoStayed = person => {
+    let newPerson = true;
+    if (uniquePeople.includes(person)) {
+      uniquePeople.splice(uniquePeople.indexOf(person), 1);
+      newPerson = !newPerson;
+    } else if (uniquePeople.indexOf(person) === -1 && newPerson) {
```

Comment:
you do includes - once and indexOf - twice, call indexOf once and use it everywhere